contrib/endpoints/src/api_manager/BUILD

```diff
@@ -164,7 +174,7 @@ cc_test(
     ],
     linkstatic = 1,
     deps = [
-        ":api_manager",
+        ":http_template",
```

Comment:
Right, `PathMatcher` and `PathMatcherBuilder` will be in next PR :) I'm trying to keep PRs as small as possible.